features/formatter/developer_is_shown_diffs.feature

```diff
@@ -193,11 +193,17 @@ Feature: Developer is shown diffs
     Then I should see:
       """
             -    'i' => 1
+      """
+    And I should see:
```

Comment:
There seems to be an "," added to the diff message in php 8. I've split the expectations out to pass in 7 and 8. Feels slightly clunky. Love to hear others' opinions